src/components/Patient/PatientRegistration.tsx

```diff
@@ -0,0 +1,1050 @@
+                            {state.name}
+                          </SelectItem>
+                        ))}
+                      </SelectContent>
+                    </Select>
+                  </div>
+                  <div>
+                    <Select
+                      {...selectProps("district")}
+                      value={form.district?.toString()}
+                      onValueChange={(value) =>
+                        setForm((f) => ({
+                          ...f,
+                          district: Number(value),
+                          local_body: undefined,
+                          ward: undefined,
+                        }))
+                      }
+                      disabled={
+                        !form.state ||
+                        districtsQuery.isLoading ||
+                        !districtsQuery.data?.length
+                      }
+                    >
+                      <SelectTrigger
+                        label={t("district")}
+                        required
+                        errors={errors["district"]}
+                        className="w-full"
+                      >
+                        <SelectValue />
+                      </SelectTrigger>
+                      <SelectContent>
+                        {districtsQuery.data?.map((district) => (
+                          <SelectItem value={district.id.toString()}>
+                            {district.name}
+                          </SelectItem>
+                        ))}
+                      </SelectContent>
+                    </Select>
+                  </div>
+                  <div>
+                    <Select
+                      {...selectProps("local_body")}
+                      onValueChange={(value) =>
+                        setForm((f) => ({
+                          ...f,
+                          local_body: Number(value),
+                          ward: undefined,
+                        }))
+                      }
+                      disabled={
+                        !form.district ||
+                        localBodyQuery.isLoading ||
+                        !localBodyQuery.data?.length
+                      }
+                    >
+                      <SelectTrigger
+                        label={t("local_body")}
+                        required
+                        errors={errors["local_body"]}
+                        className="w-full"
+                      >
+                        <SelectValue />
+                      </SelectTrigger>
+                      <SelectContent>
+                        {localBodyQuery.data?.map((localbody) => (
+                          <SelectItem value={localbody.id.toString()}>
+                            {localbody.name}
+                          </SelectItem>
+                        ))}
+                      </SelectContent>
+                    </Select>
```

Comment:
<img width="629" alt="image" src="https://github.com/user-attachments/assets/4d39b273-9499-4a5a-a1e2-edc80a67c042" />
All of these should be a search select, its really hard to find something from this large list.